app/serializers/api/v0/enterprise_serializer.rb

```diff
@@ -0,0 +1,10 @@
+class Api::V0::EnterpriseSerializer < ActiveModel::Serializer
+  attributes :id, :url, :name, :email, :website, :category
+  attributes :description, :long_description
+
+  has_one :address
+
+  def url
+    enterprise_url(object)
```

Comment:
This is not a sensible url at the moment. Perhaps we could introduce a helper that makes this a group-url for groups and a producer-url for producers (#624), and a shop-url for a hub.